frontend/components/common/Layout/Footer.tsx

```diff
@@ -290,27 +296,37 @@ function NewsletterForm({
                   ref={inputRef}
                   disabled={subscription.status !== SubscriptionStatus.Idle}
                   placeholder={emailPlaceholder}
+                  hidden={subscription.status === SubscriptionStatus.Subscribed}
                />
+               {subscription.status === SubscriptionStatus.Subscribed && (
+                  <Text align="center">
+                     <Icon as={RiCheckFill} boxSize="5" mb="-5px" />
+                     Subscribed!
+                  </Text>
+               )}
                <FormErrorMessage>{subscription.error}</FormErrorMessage>
             </FooterNewsletterFormControl>
             <Button
                type="submit"
+               height={
+                  subscription.status === SubscriptionStatus.Subscribed
+                     ? 0
+                     : '40px'
+               }
                isLoading={
                   subscription.status === SubscriptionStatus.Subscribing
                }
-               loadingText="Subscribing"
                disabled={subscription.status !== SubscriptionStatus.Idle}
-               leftIcon={
-                  subscription.status === SubscriptionStatus.Subscribed ? (
-                     <Icon as={RiCheckFill} boxSize="5" mb="-2px" />
-                  ) : undefined
-               }
-               flexShrink={0}
                colorScheme="brand"
+               visibility={
+                  subscription.status === SubscriptionStatus.Subscribed
```

Comment:
Might want to make an "isSubscribed" const so avoid all this repetition.

Reply:
Resolved 1555566b8c7e180a75bce2e718665cc5881c5ed2